packages/core/lib/console-child.js

```diff
@@ -23,33 +23,31 @@ global.crypto = {
 const { network, config, url } = yargs(input[0]).argv;
 const detectedConfig = Config.detect({ network, config });
 
-function getGanacheUrl(customConfig) {
-  const ganacheOptions = {
+function getConfiguredNetworkUrl(customConfig) {
+  const configuredNetworkOptions = {
     host: customConfig.host || managedGanacheDefaultHost,
     port: customConfig.port || managedGanacheDefaultPort
   };
-  return `http://${ganacheOptions.host}:${ganacheOptions.port}/`;
+  return `http://${configuredNetworkOptions.host}:${configuredNetworkOptions.port}/`;
 }
 
 let configuredNetwork;
 
 if (url) {
-  // Use "url" to configure network
+  // Use "url" to configure network (implies not "develop")
   configuredNetwork = {
-    network,
     network_id: "*",
     url
   };
 } else {
   // Otherwise derive network settings
-  const customConfig = detectedConfig.networks.develop || {};
+  const customConfig = detectedConfig.networks[network] || {};
```

Comment:
Should we use `develop` as a fallback? Because the settings in `config.networks.develop` are meant for the develop console. Or do we pass the name "develop" as a network in the case when it is the development console? Like maybe it should be
```
detectedConfig.networks[network] || detectedConfig.develop || {};
```
?